src/bluesky_stomp/messaging.py

```diff
@@ -214,6 +225,7 @@ def _send_bytes(
         logging.info(f"SENDING {message} to {destination}")
 
         headers: dict[str, Any] = {"JMSType": "TextMessage"}
+
```

Comment:
Need tracing on the send/send_and_receive methods so that we can see the message that is trying to be sent before it gets converted to just bytes along with its destination. No need to bother with the correlation_id as the tracing will replace tha with the trace_id it generates.